wsl-setup

```diff
@@ -87,8 +87,10 @@ if [ -n "${local_app_data}" ]; then
   cp "/usr/share/fonts/truetype/ubuntu/UbuntuMono[wght].ttf" "${fonts_dir}"
 fi
 
-# Wait for cloud-init to finish.
-cloud-init status --wait > /dev/null 2>&1 || true
+# Wait for cloud-init to finish if systemd and its service is enabled.
+if `systemctl is-enabled --quiet cloud-init.service 2>/dev/null`; then
```

Comment:
Thanks for checking! I waned to ask you to test it as it’s not easy to test in a real distribution. I only tried to prevent dbus to answer which was handling this.